src/main/scala/Utilities.scala

```diff
@@ -0,0 +1,40 @@
+import org.apache.log4j.Level
+import java.util.regex.Pattern
+import java.util.regex.Matcher
+
+object Utilities {
```

Comment:
this naming is an anti-pattern: http://gsferreira.com/archive/2017/09/please-don-t-name-it-utils/